mocks/mocks.py

```diff
@@ -95,48 +207,41 @@ def scale_light_by_distance(particle_positions,
             particle_positions,
             particle_values,
             comoving_correction_length,
-            viewing_axis=viewing_axis)
+            viewing_axis=config['viewing_axis'])
     # Converting co-moving distances to luminosity distances.
     particle_positions = convert_to_lumonsity_distance(particle_positions,
-                                                       viewing_axis,
+                                                       config['viewing_axis'],
                                                        cosmo)
     # Applying corrections to the mass weights of particles.
     particle_values =\
         particle_values / (particle_positions[:, AxisNumber[
-                           viewing_axis]] / physical_distance) ** 2
+            config['viewing_axis']]] / config['galaxy_distance']) ** 2
 
     return particle_positions.value, particle_values
 
 
-def prepare_mocks(config_location='config_example.yaml',
-                  **kwargs):
+def prepare_mocks(config):
     """
     Creates a dictionary containing configuration data and a numpy.array of
     the simulation data.
 
     Parameters
     ----------
-    config_location : str, optional
-        The name (location) of the yaml file that contains initial
-        information, default `config_example.yaml`.
+    config : dict
+        A dictionary of configuration items.
 
     Returns
     -------
     mock_image_input: dict
     galaxy_sim_data_raw : numpy.array
         Prepared information for creating mock images and the simulation data.
-    """
-    # Loading configuration file:
-    config = load_yaml_config(config_location)
-    config['galaxy_coordinates'] =\
-        kwargs.get('galaxy_coordinates',
-                   config.get('galaxy_coordinates',
-                              '14h40m56.435s -60d53m48.3s'))
-    config['observation_time'] =\
-        kwargs.get('observation_time',
-                   config.get('observation_time',
-                              '2018-04-12T08:00'))
 
+    Deleted Parameters
```

Comment:
these should be removed I think

Reply:
I see you resolved this, which I guess means its OK to leave these. Let me know in slack why you include Deleted Parameters -- I've never heard of this and want to learn.